Extensions/alternate_audio_player.js

```diff
@@ -0,0 +1,60 @@
+//* TITLE Alternate Audio Player **//
```

Comment:
Would be cool to title this "simple audio player" to match the similar "simple video player"